Clarify content-type requirements (#1344)
meilisearch/documentation: reference/api/README.md

#### Content type

> Any endpoint with a payload (`--data-binary`) requires a `Content-Type` header. MeiliSearch currently supports the following formats:

Author: ```suggestion
Any API request with a payload (`--data-binary`) requires a `Content-Type` header. MeiliSearch currently supports the following formats:
```

- `Content-Type: application/json` for JSON
- `Content-Type: application/x-ndjson` for NDJSON
- `Content-Type: text/csv` for CSV

> However, the only endpoints which currently accept NDJSON and CSV `Content-Type` are the [add documents](/reference/api/documents.md#add-or-replace-documents) and [update documents](/reference/api/documents.md#add-or-update-documents) endpoints. For all others, use `Content-Type: application/json`.

Reviewer: Nitpick: I feel this "However" only makes the sentence more complex without providing any benefits.

```suggestion
The only endpoints which currently accept NDJSON and CSV `Content-Type` are the [add documents](/reference/api/documents.md#add-or-replace-documents) and [update documents](/reference/api/documents.md#add-or-update-documents) endpoints. For all others, use `Content-Type: application/json`.
```

> However, the [add documents](/reference/api/documents.md#add-or-replace-documents) and [update documents](/reference/api/documents.md#add-or-update-documents) endpoints are the only ones which currently accept NDJSON and CSV `Content-Type`. For all other endpoints, use `Content-Type: application/json`.

Author: ```suggestion
However, the only endpoints that currently accept NDJSON and CSV `Content-Type` are the [add documents](/reference/api/documents.md#add-or-replace-documents) and [update documents](/reference/api/documents.md#add-or-update-documents) endpoints. For all others, use `Content-Type: application/json`.
```
